src/styles/pages/_reading-room.scss

```diff
@@ -38,6 +38,6 @@
 	margin: $spacer-md 0 auto;
 
 	@include respond-at("md") {
-		margin: 17rem 0 auto;
+		margin: 12vh 0 auto;
```

Comment:
Geen fan van fluid values, liever meer breakpoints en consistentie doorheen de app in de unit gebruikt voor witruimte

Reply:
veranderd naar `auto` om verticaal te centreren, ik was met die value aan het spelen maar wrs niet terug gezet 😄